salt/modules/file.py

```diff
@@ -5732,6 +5825,8 @@ def check_file_meta(
 
     if not lstats:
         changes["newfile"] = name
+        if any([ignore_ordering, ignore_whitespace, ignore_comment_characters]):
+            return True, changes
```

Comment:
This is returning a tuple, while it previously would just return the changes dictionary. This might cause issues when this function returns what is not expected.

Reply:
@Ch3LL , I checked for usage within the salt code and it won't cause issues there. how would you like me to proceed?